project/src/main/world/cutscene-manager.gd

```diff
@@ -86,7 +86,7 @@ Transitions to the cutscene at the front of the queue, staying at the current le
 """
 func replace_cutscene_trail() -> void:
 	if not is_front_chat_tree():
-		push_error("CutsceneManager._queue.front (%s) is not a cutscene" % [_queue.front()])
+		push_warning("CutsceneManager._queue.front (%s) is not a cutscene" % [_queue.front()])
```

Comment:
This feels better suited as an error. Not loading a cutscene is very bad and could make the game softlock.

Reply:
(Both of the errors in this file.)

Reply:
Reverted.